Package.swift

```diff
@@ -1,15 +1,38 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.3
 import PackageDescription
 
 let package = Package(
     name: "jaeger-client-swift",
     products: [
         .library(name: "Jaeger", targets: ["Jaeger"]),
+        .library(name: "ZipkinRecordingStrategy", targets: ["ZipkinRecordingStrategy"]),
+    ],
+    dependencies: [
+        .package(name: "swift-context", url: "https://github.com/slashmo/gsoc-swift-baggage-context", from: "0.5.0"),
+        .package(url: "https://github.com/slashmo/gsoc-swift-tracing.git", .branch("main")),
+        .package(url: "https://github.com/slashmo/swift-w3c-trace-context.git", from: "0.4.0"),
+        .package(url: "https://github.com/slashmo/swift-nio.git", .branch("feature/baggage-context")),
     ],
     targets: [
-        .target(name: "Jaeger"),
+        .target(name: "Jaeger", dependencies: [
+            .product(name: "Tracing", package: "gsoc-swift-tracing"),
+            .product(name: "W3CTraceContext", package: "swift-w3c-trace-context"),
+            .product(name: "Baggage", package: "swift-context"),
+            .product(name: "NIO", package: "swift-nio"),
+            .product(name: "NIOConcurrencyHelpers", package: "swift-nio"),
+        ]),
         .testTarget(name: "JaegerTests", dependencies: [
             .target(name: "Jaeger"),
+            .product(name: "NIOInstrumentation", package: "gsoc-swift-tracing"),
```

Comment:
FYI i think we'll want to put this somewhere else rather by the way, we may need new repos because dependencies.

Reply:
> FYI i think we'll want to put this somewhere else rather by the way, we may need new repos because dependencies.
> @ktoso

Thanks for the heads-up, shouldn't involve major changes in this repo though 👍

Reply:
> JaegerZipkinExporter?
> @ktoso

See [comment above](https://github.com/slashmo/jaeger-client-swift/pull/1#discussion_r502337662) 👆